src/v/iceberg/manifest_file_packer.h

```diff
@@ -0,0 +1,26 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+#pragma once
+
+#include "container/fragmented_vector.h"
+#include "iceberg/manifest_list.h"
+
+namespace iceberg {
+
+class manifest_packer {
+public:
+    using bin_t = chunked_vector<manifest_file>;
+    // Simple bin-packer for manifest files. Bins are packed back to front.
+    // This allows manifest lists to be prepended to and bin-packed repeatedly
+    // without re-packing the back of the list.
```

Comment:
yeh i think maybe a comment about the semantics of the output (given the reversing etc... that is in the impl of this pack)

Reply:
Done